src/FSharp.Formatting.ApiDocs/GenerateModel.fs

```diff
@@ -1019,10 +1048,16 @@ type internal CrossReferenceResolver(root, collectionName, qualify, extensions)
                 | true, (:? FSharpEntity as entity) ->
                     let urlBaseName = getUrlBaseNameForRegisteredEntity entity
 
-                    Some
+                    // See if we find the member that was intended, otherwise default to containing entity
+                    tryGetShortMemberNameFromMemberName memberName
+                    |> Option.bind (fun shortName ->
+                        findIList (entity.MembersFunctionsAndValues) (fun mfv -> mfv.DisplayName = shortName))
```

Comment:
`findIList` can be replaced with:
```fsharp
entity.MembersFunctionsAndValues |> Seq.tryFind (fun mfv -> mfv.DisplayName = shortName)
```

Reply:
That's the spell I was looking for. Thank you